### DISCUSSION TITLE
WWDC23 위너 Organization 초대 안내

### DISCUSSION THREAD

**Discussion by x-0o0** [wwdc-kr/submissions#13] [초대요청]
🎉 WWDC23 Swift Student Challenge 에 위너로 선발되신 것을 축하드립니다

WWDC Scholars Korea 에 초대되지 않으신 분들은 현재 discussion에 이메일을 댓글로 남겨주세요.

위너 선발 여부 확인 후 바로 초대 됩니다.

**Comment by yongbeomkwak** [comment]
yongbeomkwak@gmail.com 입니다 !

**Reply by x-0o0** [reply]
메일 드렸습니다. 확인 부탁드려요.

**Comment by mosiccan** [comment]
mosiccan@gmail.com 입니다!

**Reply by x-0o0** [reply]
메일 드렸습니다. 확인 부탁드려요.

**Comment by byoth** [comment]
안녕하세요. taehyunjo97@gmail.com 입니다. 확인 부탁드립니다!

**Reply by x-0o0** [reply]
메일 드렸습니다. 확인 부탁드려요.

**Comment by JUNY0110** [comment]
안녕하세요. wnsdyds403@gmail.com입니다. 확인 부탁드립니다

**Reply by x-0o0** [reply]
메일 드렸습니다. 확인 부탁드려요.

**Comment by a-mystic** [comment]
안녕하세요. 너무 늦게 발견해서 이제서야 작성하게 되었습니다. 혹시 지금도 가능할까요? totocaca@naver.com 입니다.

**Reply by x-0o0** [reply]
메일 드렸습니다. 확인 부탁드려요.

**Comment by ge-um** [comment]
안녕하세요, 이런 게 있는 줄 늦게 알아서 댓글 남깁니다. gageum0@gmail.com 입니다!

**Reply by x-0o0** [reply]
메일 드렸습니다. 확인 부탁드립니다.

**Comment by ge-um** [comment]
네 초대 주셔서 감사합니다! 메일 받았고 카카오 오픈채팅방이랑 깃허브까지 다 확인했습니다. 감사합니다!
